src/components/ProductItem/ProductItem.js

```diff
@@ -38,12 +38,12 @@ class ProductItem extends Component {
   }
 
   get productStatus() {
```

Comment:
Let's work with @nnnnat and @aldeed to see if we can move all of this logic out of the client and publish an inventory status to the catalog instead.